src/deluge/dsp/reverb/reverb.hpp

```diff
@@ -103,5 +110,6 @@ class Reverb : reverb::Base {
 	float damping_;
 	float width_;
 	float hpf_;
+	float lpf_;
```

Comment:
This isn't initialized before it's used by setModel to set the lpf frequency - that's probably why changing the initializer values doesn't work